pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -828,4 +846,9 @@ public static void verifyHybridTableConfigs(String rawTableName, TableConfig off
           rawTableName, offlineTimeColumnName, realtimeTimeColumnName));
     }
   }
+
+  // enum of all the skip-able validation types.
+  public enum ValidationType {
+    ALL, TASK, INDEX, INGEST
```

Comment:
should we allow skipping INGEST? How about we add more modes (other than TASK) only when the requirement arises?

Reply:
wilco. actually jackie requested the type ALL but yeah for now only ALL and TASK will be there.